src/app/shared/components/AnnualBackersIncentives/AnnualBackerIncentives.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  useGetABIFromChain,
+  useGetMetricsAbiWithGraph,
+  useGetMetricsAbiWithStateSync,
+} from '@/app/collective-rewards/shared'
+import { useHandleErrors } from '@/app/collective-rewards/utils'
+import { CommonComponentProps } from '@/components/commonProps'
+import { Metric } from '@/components/Metric/Metric'
+import { LoadingSpinner } from '@/components/LoadingSpinner/LoadingSpinner'
+import { Typography } from '@/components/TypographyNew/Typography'
+import Big from 'big.js'
+import { withFallbackRetry } from '@/app/shared/components/Fallback/FallbackWithRetry'
+import { ErrorBoundary } from 'react-error-boundary'
+import { useFeatureFlags } from '@/shared/context/FeatureFlag'
+import { FC } from 'react'
+
+interface AnnualBackerIncentivesProps {
```

Comment:
Not sure about this approach

Reply:
I like it, we can make it more generic and include something like `data: T`, (but we can use a name different than data)

Reply:
I like the new approach